ext/vscode/README.md

```diff
@@ -5,6 +5,9 @@ This extension makes it easier to run, create Azure Resources, and deploy Azure
 ## What It Does
 For more information about Azure Developer CLI and this VS Code extension, please [see the documentation](https://aka.ms/azure-dev/vscode).
 
+## Extension Documentation
+- [Available Commands](docs/commands.md) - Documentation for commands provided by this extension, including utility commands for use in VS Code configurations.
```

Comment:
This sentence kinda implies that _all_ of the commands are documented, which is not the case.